langserve/playground/src/components/StreamOutput.tsx

```diff
@@ -0,0 +1,108 @@
+import { str } from "../utils/str";
+
+// inlined from langchain/schema
+interface BaseMessageFields {
+  content: string;
+  name?: string;
+  additional_kwargs?: {
+    [key: string]: unknown;
+  };
+}
+
+class AIMessageChunk {
```

Comment:
Do we need to handle FunctionMessageChunk later?

Reply:
Good catch, need to handle FunctionMessageChunk as well